### ISSUE TITLE
ProhibitBitwiseOperators should not use "core" theme

### ISSUE THREAD

**Issue by otrosien** [petdance/perl-critic-bangs#16]
"core" is reserved for Perl::Critics own policies.

**Comment by wbraswell** [comment]
@otrosien WOW how did you beat me to the punch on this one?!?

@petdance This is causing improper test failures in RPerl:

https://github.com/wbraswell/rperl/issues/28

As @thaljef has recently stated, and @otrosien has correctly re-iterated, the "core" theme is reserved:

```
That's a problem with the policy module, not Perl::Critic itself. The author has mistakenly declared that it is a "core" module.
```

https://github.com/Perl-Critic/Perl-Critic/issues/712

**Comment by otrosien** [comment]
@wbraswell it turned up when I worked on https://github.com/Perl-Critic/Perl-Critic/pull/719 cause I had Perl::Critic::Bangs installed locally. :)

**Comment by petdance** [comment]
So what should the fix be?

**Comment by wbraswell** [comment]
All the other policies in this distro seem to use:

```
sub default_themes       { return qw( bangs maintenance ) }
```

https://github.com/petdance/perl-critic-bangs/blob/dev/lib/Perl/Critic/Policy/Bangs/ProhibitCommentedOutCode.pm#L24

So I would assume the line above can replace the incorrect line in ProhibitBitwiseOperators:

```
sub default_themes       { return qw( core bugs )        }
```

https://github.com/petdance/perl-critic-bangs/blob/dev/lib/Perl/Critic/Policy/Bangs/ProhibitBitwiseOperators.pm#L22

**Comment by wbraswell** [comment]
Basically, any fix where "default_themes()" no longer returns "core"...

**Comment by petdance** [comment]
Fixed with e9c072d.  Will go out in 1.11_03.

**Comment by wbraswell** [comment]
:+1:
